qemu/tests/cfg/nvdimm.cfg

```diff
@@ -92,3 +93,13 @@
         - nvdimm_hotplug:
             del mem_devs
             target_mems = mem1
+        - nvdimm_nvml:
+            depends_pkgs = ndctl-devel daxctl-devel pandoc
+            nvml_test = yes
+            nvml_dir = /tmp/nvml
+            get_nvml = "git clone --depth=1 https://github.com/pmem/nvml.git ${nvml_dir}"
+            config_nvml = "cd ${nvml_dir}; make -j4"
```

Comment:
Only compile, no install ?

Reply:
We don't need to install to run tests.

Reply:
So, the polarion test case step need be updated, right ?